tests/core/v5_1/test_v51_rpc_handlers.py

```diff
@@ -129,6 +129,28 @@ async def test_v51_rpc_ping_web3(make_request, bob_node_id_param_w3, alice, bob,
     assert pong.packet_port == alice.endpoint.port
 
 
+@pytest.mark.trio
+async def test_v51_rpc_send_ping(make_request, bob_node_id_param):
+    response = await make_request("discv5_sendPing", [bob_node_id_param])
+    assert response["request_id"] > 0
+
+
+@pytest.mark.trio
+async def test_v51_rpc_send_ping_invalid_node_id(make_request, invalid_node_id):
+    with pytest.raises(Exception, match="'error':"):
+        await make_request("discv5_sendPing", [invalid_node_id])
+    with pytest.raises(Exception, match="'error':"):
+        await make_request("discv5_sendPing", [])
```

Comment:
Since `invalid_node_id` is a parametrized fixture, I think that this test will get run multiple times.  I would recommend shifting it out into it's own test.